src/App.tsx

```diff
@@ -1,11 +1,14 @@
-import './App.css'
+import ALLmeeting from './Pages/Allmeeting'
+
 
 function App() {
 
   return (
     <>
+    
+    < ALLmeeting />
```

Comment:
This is not the right way to add your page to the app. Could you use routes?
@jsax4life already added the router, you can add a link to the page you created on the router.